src/components/User.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+export default function User({ history, data }) {
+  console.log("sda", history);
+  const handleLink = (data) => {
+    console.log("pressed");
+    console.log(data.name.first);
+    history.push({
```

Comment:
`history.push({
      pathname: `/${data.name.first}`,
      state: {
        fname: data.name.first,
        lname: data.name.first,
        email: data.email,
        phone: data.phone,
        picture: data.picture.large,
      },
    });`

Change above to:

`history.push(`/${data.name.first}`, { data });`